### DISCUSSION TITLE
Combinging scATAC + Multiome

### DISCUSSION THREAD

**Discussion by annrosebright** [GreenleafLab/ArchR#1597] [Questions / Documentation] [answered]
Hi,

We have been using ArchR for analyzing our scATAC seq data. Recently we also have generated Multiome data, which we would like to combine with the scATAC seq data. For this, I combined all the arrow files( scATACseq and Multiome) to make the ArchR object. Then I tried adding the gene expression data of the Multiome in the following way:
proj_B_kb01c <- addGeneExpressionMatrix(input = proj_B_kb01c, seRNA = seRNA, force = TRUE, threads = 1, strictMatch = F)

It runs fine where the transcriptome has equivalent cells but gives an error as soon as it is the cell from the scATACseq cells. 
1) Is there any way to avoid this? Or is there any other way where we could merge scATACseq data with Multiome data? 

For our analysis, it would have been great if we could merge our previous datasets with Multiome.

Best,
Ann

 
[ArchR-addGeneExpressionMatrix-19cb818686965-Date-2022-08-29_Time-18-27-44.log](https://github.com/GreenleafLab/ArchR/files/9446373/ArchR-addGeneExpressionMatrix-19cb818686965-Date-2022-08-29_Time-18-27-44.log)

**Comment by rcorces** [comment, marked as answer]
In principle, no. This approach is just not analytically easy to support and has not been implemented yet.
If you want to look at multiomic data, your project should only contain cells with multiomic data. You can, of course, only look at ATAC-seq and then have a project dedicated to all cells.

**Comment by annrosebright** [comment]
Hi, 

Thank you very much.

Another question I have is regarding the difference between Tilematrix and Peakmatrix. I understand that Tilematrix consists of binned signals from the entire genome, in which gene body, enhancers, and promoters are included.  Does the Peakmatrix also consistent with signals from Gene bodies in addition to Promoters, and Enhancers?  I wanted to use Peakmatrix for addIterativeLSI with only signals from promoters and enhancers. So was wondering if that's the case.
Is there any way I could remove the gene body accessibility signal from the peak matrix? 

Best,
Ann

**Reply by rcorces** [reply]
PeakMatrix is based on the peak set which is defined based on peak calling. Of course, intronic regulatory elements are common and therefore your peak set will include intronic ("gene body") peaks. You can define the peak set using `addPeakSet()`. If you wanted, you could filter your peak set to remove any regulatory elements in the gene body and then re-add that new peak set using `addPeakSet()` and then calling `addPeakMatrix()` again.
